content/tokio/tutorial/shared-state.md

```diff
@@ -90,7 +90,7 @@ async fn main() {
 # async fn process(_: tokio::net::TcpStream, _: Db) {}
 ```
 
-## On using `std::sync::Mutex`
+## Most of the time, don't use `std::sync::Mutex`
```

Comment:
Yeah, the title I suggested is not correct.
What I intended to write, was`Most of the time, don't use `tokio::sync::Mutex` which is better aligned with the content of this segment.

The motivation for this wording, was to highlight the possibly surprising fact, that the async Mutex that comes with Tokio, is often not the best pick with Tokio.

I think your suggestion is an improvement to the original version, and obviously better, than what was proposed in this PR. So I think it would be a fine addition.

Reply:
The title is changed to what @hds suggests in the latest commit. 🙏